android/src/main/java/com/swmansion/rnscreens/Screen.kt

```diff
@@ -371,6 +372,20 @@ class Screen(
         }
     }
 
+    private fun isInsideScrollViewWithRemoveClippedSubviews(child: ViewGroup): Boolean {
+        if (child is ReactHorizontalScrollView || child is ReactScrollView) {
+            return false
+        }
+        var parentView = child.parent
+        while (parentView is ViewGroup && parentView !is ScreenStack) {
```

Comment:
It might be enough to check for `Screen`, right? `Screen` will be direct descendant of `ScreenStack` and we don't want to look further up. Am I right here?

```suggestion
        while (parentView is ViewGroup && parentView !is Screen) {
```

Reply:
The loop actually finishes quicker with this check for `ScreenStack`. It's because the `ActionMenuView` goes through the loop as well and there's no `Screen` between it and the `ScreenStack`. I didn't want to add yet another check for that.